library/std/src/sys/pal/windows/fs.rs

```diff
@@ -373,12 +373,19 @@ impl File {
                     reparse_tag = attr_tag.ReparseTag;
                 }
             }
+
+            // FILE_BASIC_INFO contains additional fields not returned by GetFileInformationByHandle
+            let basic_info = self.basic_info()?;
```

Comment:
I don't know what the failure conditions are here, is accessing `FILE_BASIC_INFO` expected to never fail? Looks like uwp already does this.

If failures are possible, you can just do `self.basic_info().ok().map(|info| c::FILETIME { ... })`.